quotes.json

```diff
@@ -2523,6 +2529,18 @@
     "author": "Friedrich von Schiller",
     "text": "If you want to study yourself, look into the hearts of other people. If you want to study other people, look into your own heart."
   },
+  {
+    "author": "Fyodor Dostoyevsky",
+    "source": "https://www.goodreads.com/author/quotes/3137322.Fyodor_Dostoyevsky",
+    "tags": "pain, suffering, life",
+    "text": "Pain and suffering are always inevitable for a large intelligence and a deep heart. The really great men must, I think, have great sadness on earth."
+  },
```

Comment:
Again, this quote is not very inspiring or positive. It might be `true` that `some` great men have sadness, but at the same time there's no empirical evidence for it. It's just not logical. Go through this list: https://en.wikipedia.org/wiki/The_100:_A_Ranking_of_the_Most_Influential_Persons_in_History do _all_ of them strike you as having the "great sadness on earth" ? 🤷 
```suggestion
```